examples/walkthrough/k8s201.md

```diff
@@ -48,15 +91,35 @@ spec:
             - containerPort: 80
 ```
 
-### Services
+To create such a replication controller:
```

Comment:
Depending on how you format the pod management section in 101, this (lines 96-110) section (and also the similar "services" section, lines 137-166) should be formatted similarly.

In the 101 page i suggested to highlight these task and clarify them as "common key tasks" because as they are now, they are not introduced.